doc/source/tune/tutorials/tune-storage.rst

```diff
@@ -281,7 +279,7 @@ Again, we're running this example script from the Ray cluster's head node.
         my_trainable,
         run_config=air.RunConfig(
             name="my-tune-exp",
-            local_dir="/tmp/mypath",
+            storage_path="/tmp/mypath",
             # Use the default syncing behavior
             # You don't have to pass an empty sync config - but we
             # do it here for clarity and comparison
```

Comment:
We can remove this empty `SyncConfig`. We don't pass it in anymore in the examples above.